ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -404,17 +434,47 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
         let trailingAccessoryAreaWidth = Self.labelTrailingAccessoryAreaWidth(viewWidth: trailingAccessoryViewWidth,
                                                                               text: text, labelAccessoryViewMarginLeading: labelAccessoryViewMarginLeading)
 
-        let availableWidth = textAreaWidth - (leadingAccessoryAreaWidth + trailingAccessoryAreaWidth)
+        let availableWidth = textAreaWidth - (leadingAccessoryAreaWidth + trailingAccessoryAreaWidth + labelAccessoryViewMarginTrailing)
+        if isAttributedTextSet, let attributedText = attributedText {
+            return preferredLabelSize(with: attributedText, availableTextWidth: availableWidth)
+        }
         return text.preferredSize(for: font, width: availableWidth, numberOfLines: numberOfLines)
     }
 
+    private static func preferredLabelSize(with attributedText: NSAttributedString,
+                                           availableTextWidth: CGFloat = .greatestFiniteMagnitude) -> CGSize {
+        // We need to have .usesDeviceMetrics to ensure that there is no trailing clipping in our label.
+        // However, it causes the bottom portion of the label to be clipped instead. Creating a caluclated CGRect
+        // for width and height accommodates for both scenarios so that there is no clipping.
+        let estimatedBoundsRectForWidth = attributedText.boundingRect(
+            with: CGSize(width: availableTextWidth, height: .greatestFiniteMagnitude),
+            options: [.usesLineFragmentOrigin, .usesFontLeading, .usesDeviceMetrics],
+            context: nil)
+        let estimatedBoundsRectForHeight = attributedText.boundingRect(
+            with: CGSize(width: availableTextWidth, height: .greatestFiniteMagnitude),
```

Comment:
instead of availableWidth: should it be the first estimatedBoundRect's width? then you don't need to do the max() calculation on line 459?

Reply:
scratch that. i think you still might be having clipping issues on landscape mode

Reply:
`        let estimatedBoundsHeight = attributedText.boundingRect(
            with: CGSize(width: availableTextWidth, height: .greatestFiniteMagnitude),
            options: [.usesLineFragmentOrigin, .usesFontLeading],
            context: nil)

        return CGSize(width: ceil(availableTextWidth), height: ceil(estimatedBoundsHeight.height))`  
might be worth going to this route for now and comeback to it to investigate further later.